### ISSUE TITLE
upload of quizes through backend

### ISSUE THREAD

**Issue by tillmo** [tillmo/two_tiered_quiz#26]
There should be a possibility to upload a structured quiz with questions directly into the backend database, by issuing a command on the server, or by calling the RESTful API. This can also be used to populate the database with initial data, see #20. The structure should be one text for a quiz with all its questions, answers, justifications and explanations, such that there is no need to work with IDs for referencing. The text should have the following form:
```
quiz: name of
d: description of quiz
q: question 1
a: answer 1*
j: justification 1
e: explanation 1
j: justification 2*
a: answer 2
...
q: question 1
...
```
Correct answers and justifications are marked with a `*`.

**Comment by tillmo** [comment]
changed from json to structured text, because this can be written more easily. Processing it should be easy, too.

**Comment by tillmo** [comment]
fixed by #57
